agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -224,6 +227,13 @@ private static boolean isRequest(
     }
   }
 
+  private static boolean checkIsPreAggregated(SpanData span) {
```

Comment:
(do we have a way to differentiate this kind of "standard metric" vs cpu/memory metrics which I think are also referred to as "standard metrics"?)
```suggestion
  private static boolean checkIsStandardMetric(SpanData span) {
```

Reply:
let's go with "preAggregatedStandardMetric" everywhere